app/views/orders/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class="row form-header">
+  <div class="col-md-8 col-md-offset-2">
+    <h1> Order Fulfillment </h1>
+    <h3>Total Revenue: <%= number_to_currency(@ship_total.sum + @paid_total.sum) %></h3><br>
```

Comment:
This math would be good in the controller or model where the `@ship_total` and `@paid_total` variables are set